assets/scss/base.scss

```diff
@@ -433,3 +433,7 @@ a.link {
 .v-navigation-drawer--close {
   visibility: inherit;
 }
+
+.colour-dk-text {
```

Comment:
"colour" is proper British English but CSS uses the American "color"... I recommend you use that here.

Reply:
My code editor converted my spelling of this word :) I was an adamant 'colour' speller